spec/controllers/school_resgistrations_controller_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+RSpec.describe Devise::RegistrationsController, type: :controller do
+  FLASH_UPDATE_MSG = "Your account has been updated successfully."
```

Comment:
Looks like it's only being used in one place, why are you putting it in a constant?